docs/features/decorators.rst

```diff
@@ -87,3 +87,86 @@ Arbitrary number versions can be passed as parameters. On second test both RHEL
 This decorator is used to avoid false failures when an feature is supported
 only on one os version. For example, ostree repository is available
 in RHEL7 but not in RHEL6.
+
+tier[n]
+---------
+
+This set of decorators defines test levels:
+
+* ``tier1`` marks component level functional tests (that verify defined functional requirements using a range of normal and erroneous input data). Example::
+
+    from robottelo.decorators import tier1
+
+    @tier1
+    def test_positive_create_with_username(self):
+        """Create User for all variations of Username"""
+
+* ``tier2`` marks integration level functional tests and may include basic non-functional tests (security, performance regression, installation, compose validation). Example::
+
+    from robottelo.decorators import tier2
+
+    @tier2
+    def test_positive_view_cve(self):
+        """View CVE number(s) in Errata Details page"""
+
+* ``tier3`` marks system level tests::
+
+    from robottelo.decorators import tier3
+
+    @tier3
+    def test_positive_sync_with_enabled_notification(self):
+        """Receive email after every sync operation"""
+
+* ``tier4`` marks complex and long running tests. Example::
+
+    from robottelo.decorators import tier4
+
+    @tier4
+    def test_positive_upload_to_satellite(self):
+        """Perform end to end oscap test and upload reports"""
+
+skip_if_not_set
+---------------
+
+``skip_if_not_set`` skips test if one or more specified configuration options is not set in ``robottelo.properties``. It is used to define tests specific to a selected (optional) feature. Without the decorator, such tests would fail if the tested feature is not enabled. Example::
+
+    from robottelo.decorators import skip_if_not_set
+
+    @skip_if_not_set('ldap')
+    def test_positive_ldap_auth_usergroup_user_add(self):
+        """New user added to UserGroup inherits roles"""
+
+cacheable
+---------
+
+``cacheable`` makes an optional object cache available. This is used when creating factory objects for CLI tests. For example::
+
+    from robottelo.decorators import cacheable
+
+    @cacheable
+    def make_role(options=None):
+        """create a role using ``hammer role create``"""
+
+run_only_on
+-----------
+
+``run_only_on`` skips test based on a server mode. It takes one parameter that is either 'sat' (to define Satellite specific tests) or 'sam' (to define tests specific to Subscription Asset Manager). For example::
```

Comment:
Looks like this is deprecated @omaciel can you confirm?
I found no reference to `run_only_on('sam')` in the code, are running builds against sam thing?